@uportal/esco-content-menu/src/components/Ellipsis.vue

```diff
@@ -4,42 +4,63 @@
 
 <script>
 // Code translation from https://github.com/jypblue/vue-ellipsis
-// intregration due to a problem : https://github.com/jypblue/vue-ellipsis/issues/2
+// intregration due to some problems and customization needs : https://github.com/jypblue/vue-ellipsis/issues/2
 
 export default {
   name: 'Ellipsis',
   props: {
     message: {type: String, default: ''},
-    lineClamp: {type: Number, default: 1},
+    lineClamp: {type: Number, default: 0},
     lineHeight: {type: String, default: '22px'},
     endChar: {type: String, default: '...'},
     endHtml: {type: String, default: ''},
   },
   mounted() {
     this.$nextTick(function() {
-      setTimeout(() => {
-        this.handleSubstrSentence();
-      }, 200);
+      window.addEventListener('resize', this.timedRun);
+      this.timedRun();
     });
   },
   methods: {
+    timedRun() {
+      setTimeout(() => {
+        this.handleSubstrSentence();
+      }, 200);
+    },
     handleSubstrSentence() {
+      /* eslint-disable */
       const stNode = this.$refs.sentence;
       const html = this.message;
-      if (html.length === 0) {
-        // throw new Error('the String is empty');
+      if (html && html.length <= 0) {
+        // throw new Error('the String is empty');;
         return false;
       }
+      const stNodeLineHeight = parseInt(this.lineHeight, 10);
+      // we need to obtain firtly the real space available
+      stNode.innerHTML = '';
+      stNode.style.height = '100%';
+      const availableNodeHeight =
+        stNode.getBoundingClientRect().height || stNodeLineHeight;
```

Comment:
This is querying the height of the parent, correct?
Would it be easier to directly query the value, and avoid two layout reflows to determine `100%` vs `auto`?

Reply:
I can't do that !

Reply:
An alternative would be to check if there isn't collision between next element and if the element isn't ouside of the parent. I don't know what is the best and if there is an other way to purpose an ellipsis on several lines that could fit the content to the div size ?